test/fixtures/photos.yml

```diff
@@ -16,3 +16,10 @@ two:
   height: 3
   width: 3
 
+three:
+  repository_id: 3
+  image_file_name: p2
+  image_content_type: .png
+  image_file_size: 3
+  height: 3
+  width: 3
```

Comment:
I think content_type might be something like 'image/png' or something (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type or https://en.wikipedia.org/wiki/Media_type